addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -23,8 +23,12 @@
                         </t>
                     </div>
                     <div class="o_carousel_product_card_footer card-footer d-flex align-items-center">
-                        <div class="d-block font-weight-bold" t-esc="data['price']"
-                            t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
+                        <div>
+                            <span class="font-weight-bold" t-esc="data['price']"
+                                t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
+                            <del t-attf-class="text-danger ml-1 h6 {{'' if data['has_discounted_price'] else 'd-none'}}" style="white-space: nowrap;"
```

Comment:
```suggestion
                            <del t-if="data['has_discounted_price']" class="text-danger ml-1 h6" style="white-space: nowrap;"
```
Any reason not to hide it with a regular `t-if` ?

Reply:
No particular reason. Just a copy-paste...